source/slang/slang-ir-autodiff-primal-hoist.h

```diff
@@ -236,6 +236,34 @@ namespace Slang
         Dictionary<IRInst*, InversionInfo> invInfoMap;
     };
 
+    struct Use
```

Comment:
Maybe the name here could be made clearer? `UseOrPseudoUse` perhaps?

Reply:
Done.